### PR TITLE
Updates to slack connection

### PR DESCRIPTION
**Pull request by unDemian**
#### Changes proposed in this Pull Request

Fixes 3748-gh-Automattic/p2 by updating the messages and screenshots that can be seen in Calypso.

#### Testing instructions
See 3748-gh-Automattic/p2

### CODE REVIEW

File: client/my-sites/marketing/connections/service-description.jsx [hunk 1]
```diff
@@ -137,12 +137,12 @@ class SharingServiceDescription extends Component {
 			},
 			slack: function () {
 				if ( this.props.numberOfConnections > 0 ) {
-					return this.props.translate( 'Connected this P2 to your Slack Workspace.', {
+					return this.props.translate( 'Workspace connected to Slack.', {
```
Comment (a8ci18n) [inline client/my-sites/marketing/connections/service-description.jsx:140]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/my-sites/marketing/connections/service-description.jsx [hunk 2]
```diff
@@ -137,12 +137,12 @@ class SharingServiceDescription extends Component {
...
 						comment: 'Get slack notifications on new P2 posts.',
 					} );
 				}
 
-				return this.props.translate( 'Connect your Slack Workspace to this P2.', {
+				return this.props.translate( 'Connect this workspace to your Slack.', {
```
Comment (a8ci18n) [inline client/my-sites/marketing/connections/service-description.jsx:145]: :warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Connected this P2 to your Slack Workspace.' )` *ES Score: 9*

File: client/my-sites/marketing/connections/service-examples.jsx [hunk 3]
```diff
@@ -286,12 +286,35 @@ class SharingServiceExamples extends Component {
 		return [
 			{
 				image: {
-					src: '/calypso/images/sharing/slack-screenshot.png',
-					alt: this.props.translate( 'Get slack notifications on new P2 posts.', {
+					src: '/calypso/images/sharing/slack-screenshot-1.png',
+					alt: this.props.translate( 'Get Slack notifications with every new P2 post.', {
```
Comment (a8ci18n) [inline client/my-sites/marketing/connections/service-examples.jsx:290]: :warning: This change will be queued for retranslation. We'll display the English text until then.

**Review by lezama — APPROVED** [review]
looking good

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6722047

Hi @unDemian, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Workspace connected to Slack.`
- `Connect this workspace to your Slack.`
- `Get Slack notifications with every new P2 post.`
- `Get {{strong}}Slack notifications{{/strong}} with every new P2 post.`
- `Preview posts and pages directly from Slack.`
- `{{strong}}Preview posts and pages{{/strong}} directly from Slack.`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6722047).

**Pull request merged**